scripts/write_notes_to_couchdb.py

```diff
@@ -45,17 +47,48 @@ def write_note_to_couch(pid, timestamp, note, lims):
         if doc['details']['running_notes'] != json.dumps(running_notes):
             msg = 'Running note save failed from {} to {} for {}'.format(lims, config['statusdb'].get('url'), pid)
             for user_email in ['genomics-bioinfo@scilifelab.se', note['email']]:
-                email_error(msg, user_email)
+                email_responsible(msg, user_email)
+        else:
+            time_in_format = datetime.datetime.strptime(timestamp, '%Y-%m-%d %H:%M:%S.%f').strftime("%a %b %d %Y, %I:%M:%S %p")
+            subject = '[LIMS] Running Note:{}, {}'.format(pid, doc['project_name'])
+            proj_coord = '.'.join(doc['details'].get('project_coordinator','').lower().split()) + '@scilifelab.se'
+            text = 'A note has been created from LIMS in the project {}, {}! The note is as follows\n\
+            >{} - {}{}\
+            >{}'.format(pid, doc['project_name'], note['user'], time_in_format, note.get('category'), note)
+
+            html = '<html>\
+            <body>\
+            <p> \
+            A note has been created from LIMS in the project {}, {}! The note is as follows</p>\
+            <blockquote>\
+            <div class="panel panel-default" style="border: 1px solid #e4e0e0; border-radius: 4px;">\
+            <div class="panel-heading" style="background-color: #f5f5f5; padding: 10px 15px;">\
+                <a href="#">{}</a> - <span>{}</span> <span>{}</span>\
+            </div>\
+            <div class="panel-body" style="padding: 15px;">\
+                <p>{}</p>\
+            </div></div></blockquote></body></html>'.format(pid, doc['project_name'], note['user'],
+                                    time_in_format, note.get('category'), markdown.markdown(note.get('note')))
+            email_responsible(text, resp_email, error=False, subject=subject, html=html)
+
+def email_responsible(message, resp_email, error=True, subject=None, html=None):
```

Comment:
Do you think it would work to move the html creation from above into here so that the emails sent always are html formatted?

Reply:
Nah, maybe wait with that since it will need som adaption. Maybe a future improvement idea.